cpp/Esri/ArcGISRuntime/Toolkit/PopupViewController.cpp

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************
+ *  Copyright 2012-2020 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+#include "PopupViewController.h"
+
+namespace Esri
+{
+namespace ArcGISRuntime
+{
+namespace Toolkit
+{
+
+PopupViewController::PopupViewController(QObject* parent):
+  QObject(parent)
+{
+}
+
+PopupViewController::~PopupViewController()
```

Comment:
```suggestion
PopupViewController::~PopupViewController() = default
```

Reply:
Haven't been doing this one so far. I am still under the impression this didn't help build performance as dicovered by Elliot/James? Maybe I am misremembering. It is quite unusual AFAIK.

Reply:
I'm fine with either one.